File: Weekend Exercises/Java Version/TaskTwenty.java
public class TaskTwenty{

	public static void main (String [] args){

		int firstNumber = 0;
		int secondNumber = 1;

		System.out.printf("%d %d ", firstNumber, secondNumber);

		for (int counter = 1; counter <= 18; counter++){
			int number = firstNumber + secondNumber;
			firstNumber = secondNumber;
			secondNumber = number;
			
			System.out.printf("%d ", number);

		}
		System.out.println();
	}
}

File: Weekend Exercises/Python Version/task_twenty.py
first_number = 0
second_number = 1

print(f"{first_number} {second_number} ", end = "")

for value in range(18):

	next_number = first_number + second_number
	first_number = second_number
	second_number = next_number
	
	print(next_number, end = " ")

print()